modules/microshift-accessing-cluster-remotely-admin.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
```

Comment:
What is the purpose of this new module?

Reply:
To separate the procedures. It will appear seamless in the docs, this is just a structural issue.